src/plugins/utilities/link/link-fontweight.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Link font weight in paragraphs, WYSIWYG, or nested in components with open fields.
+ * Only responsive variants are available as font weight shouldn't change when hover/focus.
+ */
+module.exports = function () {
+  return function ({ addUtilities, prefix }) {
+    const utilities = {};
+
+    utilities[prefix('.link-') + 'regular'] = {
+      'a': {
+        fontWeight: '400'
+      }
+    }
+
+    utilities[prefix('.link-') + 'semibold'] = {
+      'a': {
+        fontWeight: '600'
+      }
+    }
+
+    utilities[prefix('.link-') + 'bold'] = {
+      'a': {
+        fontWeight: '700'
+      }
+    }
+
+    addUtilities(utilities, { variants: ['responsive'], respectPrefix: false })
```

Comment:
```suggestion
    utilities['.link-regular'] = {
      'a': {
        fontWeight: '400'
      }
    }

    utilities['.link-semibold'] = {
      'a': {
        fontWeight: '600'
      }
    }

    utilities['.link-bold'] = {
      'a': {
        fontWeight: '700'
      }
    }

    addUtilities(utilities, { variants: ['responsive'] })
```

You don't need to do the prefix non-sense that I had to do to get the pseudo selector prefix in front of the prefixed selector here.

Reply:
Oh yeah - thanks!